lib/render.js

```diff
@@ -80,7 +80,7 @@ export function renderRoute (url, context = {}) {
     if (!context.nuxt.serverRendered) {
       app = '<div id="__nuxt"></div>'
     }
-    const publicPath = self.options.build.publicPath.indexOf('http') === 0 ? self.options.build.publicPath : urlJoin(self.options.router.base, self.options.build.publicPath)
+    const publicPath = self.options.build.publicPath
```

Comment:
@Atinux with the current code `urlJoin(self.options.router.base, self.options.build.publicPath)` wouldn't that end up being `/my-site/app/` which is wrong?

The html files this variable is for is rendering the files generated by webpack and not handling the inner routes of the app.  So in my opinion, the `router.base` shouldn't come into play.